CMakeLists.txt

```diff
@@ -28,11 +29,13 @@ set(PROJECT_URL "https://github.com/jrl-umi3218/mc_rbdyn_urdf")
 set(PROJECT_DEBUG_POSTFIX "_d")
 
 set(DOXYGEN_USE_MATHJAX "YES")
+SET(CXX_DISABLE_WERROR True)
 setup_project()
 
 option(PYTHON_BINDING "Generate python binding." ON)
 option(PYTHON_BINDING_USER_INSTALL "Install the Python bindings in user space" OFF)
 option(DISABLE_TESTS "Disable unit tests." OFF)
+option(BENCHMARKS "Generate benchmarks." OFF)
```

Comment:
Also not needed since there is not benchmarks to compile in the project.

Reply:
This line has been removed.